bootstrap/kubeadm/main.go

```diff
@@ -204,14 +202,12 @@ func main() {
 	restConfig.Burst = restConfigBurst
 	restConfig.UserAgent = remote.DefaultClusterAPIUserAgent(controllerName)
 
-	tlsOptionOverrides, err := flags.GetTLSOptionOverrideFuncs(tlsOptions)
+	tlsOptions, metricsOptions, err := flags.GetManagerOptions(managerOptions)
 	if err != nil {
-		setupLog.Error(err, "unable to add TLS settings to the webhook server")
+		setupLog.Error(err, "Unable to compute Manager options")
```

Comment:
can there be a better message here?
i don't think the user has a context what is the "manager" here.

also Manager should likely be lowercase and the error should start with lowercase
https://github.com/kubernetes-sigs/cluster-api/blob/70da4c793705c947ff3c3853824d377a4541cfd9/bootstrap/kubeadm/main.go#L187

Reply:
What would you suggest?

Reply:
Please note that this is not an error. This is the beginning of an error log line. Log lines should start upper case

Reply:
Changed to: `Unable to start manager: invalid flags`

Reply:
If we have a problem with the word manager I would suggest a follow-up PR. We are using manager everywhere in error logs here

Reply:
ok, if manager is already in the logs then this is not a precedence.
i think that `err` is more important here so the other string can just be used to search the code base.

Reply:
Pushed the changes. Let me know if they are fine as is